internal/jimm/auditlog/auditlog_test.go

```diff
@@ -67,6 +67,67 @@ func (s *auditLogManagerSuite) Init(c *qt.C) {
 	s.user = openfga.NewUser(i3, ofgaClient)
 }
 
+func (s *auditLogManagerSuite) TestRedactSensitiveParams(c *qt.C) {
+	newEntry := func(method string) dbmodel.AuditLogEntry {
+		return dbmodel.AuditLogEntry{
+			FacadeMethod: method,
+			Params:       dbmodel.JSON(`{"params":"super secret"}`),
+		}
+	}
+
+	tests := []struct {
+		about          string
+		entry          dbmodel.AuditLogEntry
+		expectedParams dbmodel.JSON
+	}{{
+		about:          "addcredentials",
+		entry:          newEntry("AddCredentials"),
+		expectedParams: auditlog.RedactJSON,
```

Comment:
What do you think about patching the `sensitiveMethods` list and then you can just have 2 test cases, one where it redacts and one where it doesn't.

Reply:
done